packages/camera/src/components/Controls/FullScreenButton.js

```diff
@@ -1,10 +1,9 @@
+import PropTypes from 'prop-types';
 import React from 'react';
-import { useTranslation } from 'react-i18next';
 import { Text } from 'react-native';
+import { MaterialCommunityIcons } from '@expo/vector-icons';
```

Comment:
If you use `@expo/vector-icons` you need to add it to the dependencies of the package.